webapp/public/index.html

```diff
@@ -6,12 +6,38 @@
       name="viewport"
       content="minimum-scale=1, initial-scale=1, width=device-width"
     />
-    <title>EOSCR Network and Infrastructure Dashboard for EOSIO networks</title>
+    <meta httpEquiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="csrf_token" content="" />
+    <meta property="type" content="website" />
+    <meta property="url" content="%PUBLIC_URL%/" />
+    <meta name="theme-color" content="#ffffff" />
+    <meta name="_token" content="" />
+    <meta name="robots" content="noodp" />
+    <meta
+      name="title"
+      content="EOSCR Network and Infrastructure Dashboard for EOSIO networks"
```

Comment:
Can we change this for `Network Monitor and Infrastructure Dashboard for EOSIO blockchain networks`

En vez de EOSIO se podråe poner el nombre de la red para cada instancia?

Reply:
done!